packages/interface/src/components/dialog/BackupRestoreDialog.tsx

```diff
@@ -19,20 +20,23 @@ export interface BackupRestorationDialogProps {
 
 export const BackupRestoreDialog = (props: BackupRestorationDialogProps) => {
 	const platform = usePlatform();
-	const { register, handleSubmit, reset } = useForm<FormValues>({
+	const form = useForm<FormValues>({
 		defaultValues: {
 			masterPassword: '',
-			secretKey: ''
+			secretKey: '',
+			filePath: ''
 		}
 	});
 
 	const onSubmit: SubmitHandler<FormValues> = (data) => {
```

Comment:
use `form.handleSubmit` here
```suggestion
	const onSubmit = form.handleSubmit((data) => {
```
would mean u can do `onSubmit={onSubmit}` in `<form>` and u don't have to manually type it

Reply:
will do this in another PR, would be a bit too large for this one as there's at least 3/4 dialogs that need refactoring to the same standard